.gitignore

```diff
@@ -1,2 +1,3 @@
 .env
-.vscode
\ No newline at end of file
+.vscode
+node_modules/*
```

Comment:
Sorry, this is a restriction of Google Actions
node_modules must be inside an action so it could be obtained via regular git clone.
However, it is not included in `package.json`

Reply:
I didn't know that. So I'll remove this, thou!